testutil/integration/evmos/network/coins.go

```diff
@@ -28,19 +27,24 @@ type ChainCoins struct {
 // DefaultChainCoins returns the default values used for the ChainCoins in which
 // base and evm denom are the same.
 func DefaultChainCoins() ChainCoins {
-	baseCoinInfo := evmtypes.ChainsCoinInfo[utils.MainnetChainID]
-	// baseCoin is used for both base and evm coin as default..
-	baseCoin := CoinInfo{
-		Denom:    baseCoinInfo.Denom,
-		Decimals: baseCoinInfo.Decimals,
-	}
-	evmCoin := baseCoin
+	baseCoinInfo := evmtypes.ChainsCoinInfo[defaultChain]
+
+	// baseCoin is used for both base and evm coin as default.
+	baseCoin := defaultCoin(baseCoinInfo)
+	evmCoin := defaultCoin(baseCoinInfo)
 	return ChainCoins{
 		baseCoin: &baseCoin,
 		evmCoin:  &evmCoin,
 	}
 }
 
+func defaultCoin(coinInfo evmtypes.EvmCoinInfo) CoinInfo {
+	return CoinInfo{
+		Denom:    coinInfo.Denom,
+		Decimals: coinInfo.Decimals,
+	}
+}
```

Comment:
why is this need if it's a subset of the same info from the `EvmCoinInfo`?

Reply:
This is a good point, but the name of the EvmCoinInfo could be a bit misleading. Here we are referring to a generic Cosmos coin, not an EvmCoin. Probably we could refactor the EvmCoinInfo to be a generic coin type wrapper, but I would leave it for the os repo